clients/python/test/e2e/test_files_api.py

```diff
@@ -0,0 +1,43 @@
+import hashlib
+import os
+from pathlib import Path
+
+import osparc
+import pytest
+from packaging.version import Version
+
+
+def _hash_file(file: Path) -> str:
+    assert file.is_file()
+    md5 = hashlib.md5()
+    with open(file, "rb") as f:
+        while True:
+            data = f.read(65536)  # read 64kb
+            if not data:
+                break
+            md5.update(data)
+        return md5.hexdigest()
+
+
+@pytest.mark.skipif(
+    Version(osparc.__version__) < Version("0.6.0"),
+    reason=f"osparc.__version__={osparc.__version__} is older than 0.6.0",
+)
+def test_upload_file(tmp_path: Path, cfg: osparc.Configuration) -> None:
```

Comment:
@pcrespov should I skip this file until we have the `delete` operation? As it is it uploads a 10GB file.

Reply:
will do